lib/src/main/scala/spinal/lib/bus/amba4/axi/Axi4.scala

```diff
@@ -49,11 +49,16 @@ case class Axi4Config(addressWidth : Int,
                       useResp      : Boolean = true,
                       useProt      : Boolean = true,
                       useStrb      : Boolean = true,
-                      arUserWidth   : Int = -1,
-                      awUserWidth   : Int = -1,
+                      arUserWidth  : Int = -1,
+                      awUserWidth  : Int = -1,
                       rUserWidth   : Int = -1,
                       wUserWidth   : Int = -1,
-                      bUserWidth   : Int = -1) {
+                      bUserWidth   : Int = -1,
+                      
+                      readIssuingCapability     : Int = 1,
```

Comment:
We might want to put it to Int.MaxValue or -1 by default ? to say "unspecified" by default ?

Reply:
Yep, makes sense. We should use -1 by default and prevent the QSysifyEmitter from emitting the corresponding `set_interface_property` in this case.
I think that the `*Capability` parameters are optional, so Quartus will use some "safe" default. I'm not sure of this, I will check it ASAP.

Reply:
Quartus Platform Designer defaults to 1 when `*Capability` is not set in the TCL script. I will proceed as proposed in my former comment.